atests/feature_tests/combobox_listbox.robot

```diff
@@ -0,0 +1,58 @@
+*** Settings ***
+Library    WhiteLibrary
+Test Teardown    Clean Application
+Resource    ../resource.robot
+
+*** Test Cases ***
+Verify Operation Selections
+    Select Combobox Value    op    +
+    Verify Combobox Selection    op    +
+    Select Combobox Value    op    -
+    Verify Combobox Selection    op    -
+    Select Combobox Value    op    *
+    Verify Combobox Selection    op    *
+    Select Combobox Value    op    /
+    Verify Combobox Selection    op    /
+    Select Combobox Value    op    %
+    Verify Combobox Selection    op    %
+    Select Combobox Index    index=0    1    #multiplication
+    Verify Combobox Selection    op    *
+
+Combobox Selected Text
+    Select Combobox Value    op    -
+    ${selected_value}    Get Combobox Selected Text    op
+    Should Be Equal    ${selected_value}    -
+
+Combobox Contains
+    Combobox Should Contain    op    +
+    Combobox Should Contain    op    /
+    Combobox Should Contain    op    %
+    Run Keyword And Expect Error    ComboBox with locator 'op' did not contain '&'
+    ...                             Combobox Should Contain    op    &
+    Run Keyword And Expect Error    ComboBox with locator 'op' did not contain 'plus'
+    ...                             Combobox Should Contain    op    plus
+    Combobox Should Not Contain    op    &
+    Combobox Should Not Contain    op    plus
+    Run Keyword And Expect Error    ComboBox with locator 'op' should not have contained '+'
+    ...                             Combobox Should Not Contain    op    +
+
+Listbox Selected Text
+    Select Listbox Value    list_box    Teppo
+    ${selected_value}    Get Listbox Selected Text    list_box
```

Comment:
Does this work ? shouldnt it be something like `${selected_value}=`

Reply:
It works, "=" in variable assignments is optional.